src/main/pages/setup-kubernetes/kubernetes-single-user.adoc

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Single-User&#58 Deploy to Kubernetes"
+title: "Single-User&#58 Deployment to Kubernetes"
```

Comment:
Should be consistent with multi-user: https://github.com/eclipse/che-docs/pull/536/files